README.md

```diff
@@ -53,6 +53,10 @@ CONFIGURATION
 * See troubleshooting for more details.
```

Comment:
Let's add details on where to get License Key and Account ID here.
And if needed to change link on the Configuration page, let's do that as well.